charts/endpoint-controller/templates/deployment.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Would be good to add some healthchecks for endpoint-controller and define probes for the deployment